test/projects/UserManager.ts

```diff
@@ -1,8 +1,11 @@
 import {assert, driver, Key, WebElement} from 'mocha-webdriver';
 import {server, setupTestSuite} from 'test/projects/testUtils';
+import * as gu from 'test/nbrowser/gristUtils';
+
 
 describe('UserManager', () => {
   setupTestSuite();
+  gu.bigScreen();
```

Comment:
Is this needed? (I always have to adjust when I run such tests locally because bigScreen is bigger than my physical screen)